packages/MarkdownEditor-Tests.package/MarkdownEmphasisDelimiterTest.class/instance/testLength.st

```diff
@@ -1,13 +1,6 @@
 tests - state
 testLength
 
-	| delimiter |
-	delimiter := MarkdownEmphasisDelimiter newFor: (self readStreamOn: '*').
-	self assert: 1 equals: delimiter length.
-	
-	delimiter := MarkdownEmphasisDelimiter newFor: (self readStreamOn: '****').
-	self assert: 4 equals: delimiter length.
-	
-	delimiter := MarkdownEmphasisDelimiter newFor: (self readStreamOn: '*****').
-	self assert: 5 equals: delimiter length
-	
\ No newline at end of file
+	{'*'->1. '**'->2. '***'->3. '****'->4. '*****'->5} do: [:case | | delimiter |
```

Comment:
This is an alternative style I'm proposing here. If you prefer `Dictionary newFrom: {}`, I'll change it here, otherwise I would replace the old occurrences of `Dictionary newFrom:`.

------

There is also another method which tests the length, but with `calculateLength:`. I don't understand why they are necessary and test funny things like the length of a right-flanking delimiter...

https://github.com/hpi-swa-teaching/MarkdownEditor/blob/6d24bdf8863a394d9acbf6998b3ad181759eb5aa/packages/MarkdownEditor-Tests.package/MarkdownEmphasisDelimiterTest.class/instance/testLengthCalculation.st#L1-L8

Reply:
I really like that fact that using `keysAndValuesDo: [:string :length | ...]` gives the `string` and the `level` a name. Having `case key` and `case value` is really nondescriptive. 
But that doesn't work with this kind of Dictionary notation. I'd prefer to keep it like it is with the other Dictionaries.

Reply:
I don't see the problem with the `testLengthCalculation`. It tests multiple lengths to make sure that not only one length is supported (this is a perfect example of how TDD works since first it returned 1, then it supported 2 and then it supported everything else) and it also checks if it works for a closing delimiter since that is a litte different.

Reply:
Okay, after digging the code a bit more, I saw that `testLengthCalculation` only sets the `length` indirectly, so it's okay to have tests for both. But maybe this can become simpler after revisiting the `InlineTextStyler`...